orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -426,7 +455,7 @@ def _get_inference_graph(self, *inputs, **kwargs):
 
         # Setup dynamic axes for onnx model
         input_names, dynamic_axes, self._input_names_require_grad = _parse_inputs_for_onnx_export(self._original_module, *inputs, **kwargs)
-        output_names, output_dynamic_axes = _parse_outputs_for_onnx_export(self._original_module, inputs)
+        output_names, output_dynamic_axes, self._sample_user_output_type = _parse_outputs_for_onnx_export(self._original_module, inputs)
```

Comment:
I would suggest a name that resembles more to the original model, such as `self._original_module_output_type`  or maybe `self._original_module_user_output_type` which as associated with `self._original_module`